### PR TITLE
MNT: Update code examples

### PR DESCRIPTION
**Pull request by adswa**
This is a complete update of the code snippets, excluding those that had inconvenient output rendering (see #390). The changes are 

- consistent hashes
- updated dataset history
- updated command outputs, e.g., datalad status in clean datasets reporting that there is nothing to save

### CODE REVIEW

File: docs/basics/101-136-filesystem.rst [hunk 1]
```diff
@@ -603,7 +603,7 @@ section.
 
       $ cd ../mock_user && mv onemoredir/DataLad-101 .
       $ rm -r onemoredir
-      $ cd ../DataLad-101 && git reset --hard master
+      $ cd ../DataLad-101 &&  sed -i 's/..\/mock_user\/onemoredir\/DataLad-101/..\/mock_user\/DataLad-101/' .git/config
```
Comment (heytitle) [inline docs/basics/101-136-filesystem.rst]: additional space here?

Comment (adswa) [reply docs/basics/101-136-filesystem.rst]: thanks for catching that!

Comment (adswa) [inline docs/basics/101-136-filesystem.rst]:
```suggestion
      $ cd ../DataLad-101 && sed -i 's/..\/mock_user\/onemoredir\/DataLad-101/..\/mock_user\/DataLad-101/' .git/config
```

**Comment by mih** [comment]
Needs conflict resolution now.

**Comment by adswa** [comment]
done

**Comment by adswa** [comment]
There is a fair bit of maintenance in here, so I will go ahead and merge this in. :)

**Pull request merged**